### ISSUE TITLE
[Bug]: Links do not preserve the current link when opening the dialog.

### ISSUE THREAD

**Issue by KonnorRogers** [KonnorRogers/rhino-editor#214]
When opening the dialog for an element with an existing link, it _should_ show the current link text.

also, lets make the linking interface a lot nicer.

**Comment by bjhess** [comment]
So our app fixed the "preserve current link when opening dialog", and here is the code that not-me wrote in case it helps:

```js
  fixupShowLinkDialogDisplaysLinkText() {
    const rhino = this.element
    const oldShowLinkDialog = rhino.showLinkDialog.bind(rhino)

    rhino.showLinkDialog = () => {
      const attributes = rhino.editor.getAttributes('link')
      oldShowLinkDialog()
      if (attributes.href) {
        rhino.linkInputRef.value.value = attributes.href
      }
    }
  }
```

**Comment by KonnorRogers** [comment]
Fixed in 0.13.1
